function errorHandler(x) {

    try{
        
    var a=getX(x);
    
    }
    catch(err){

        console.error(err);
    }
return a;

}

function getX(x) {
    if(x == 5) {
        return new Error("Error Occured");
    }
    return x;
}

module.exports = errorHandler;
